parser/src/Parse/Module.hs

```diff
@@ -181,7 +181,11 @@ import' =
     exposing =
       do  preExposing <- try (whitespace <* reserved "exposing")
           postExposing <- whitespace
-          imports <- listing detailedListing
+          imports <-
+            choice
+              [ listing detailedListing
+              , listingWithoutParens
```

Comment:
Should we consider allowing missing parens for all listings, not just ones in the exposing clause of imports?  (I guess that's a question that can always be put off until later.)

Reply:
So, `listing` is also used for type declarations and tuples? I'd be careful, in this case.